### ISSUE TITLE
Modal malfunction on Chrome for Android when virtual keyboard is dismissed and modal closed

### ISSUE THREAD

**Issue by rob-rooam** [Temzasse/react-modal-sheet#113]
In our web application, we have identified an issue where the modal stops working under certain conditions. Specifically, this occurs when the user uses the Chrome browser on an Android device, the modal contains a field, and the virtual keyboard is dismissed. Once the modal is closed, it no longer functions properly. Looks like the main div that contains the react-modal-sheet-container and the backdrop button is not getting cleared up under this given context. It works for iOS though.

[I reproduced it in this repo](https://github.com/rob-rooam/react-modal-sheet-chrome-issue)

**Comment by immortal-blog** [comment]
Yes, even in safari for iOS, the height of this modal changes after opening and closing the virtual keyboard, becoming taller. Even destroying this component is useless. This will affect the display experience.

**Comment by JMarchandev** [comment]
same problem here, did you solve it ?
